src/index.js

```diff
@@ -72,15 +72,34 @@ const getLooseMode = looseOpts => {
   return looseOpts;
 }
 
-// TODO: Allow specifying modules as: Boolean, String or Array of module types
-const getModules = modulesOpts => {
+/**
+ * Given modulesOpts return an Array of all required module transformations
+ * @param  {String|Array.<String>|Boolean}  modulesOpts  An argument enumerating the modules to transform
+ * @return {Array.<String>}  An array of module transformation plugin names
+ */
+export const getModuleTransformations = modulesOpts => {
   if (!modulesOpts) { return []; }
-  return modulesOpts;
+
+  if (modulesOpts === true) { return Object.values(MODULE_TRANSFORMATIONS); }
```

Comment:
each module turns import -> whatever syntax (amd,cjs,systemjs,umd). You would only want to use one of the transforms or none (keep import). I'm just saying no one would want to choose multiple

(Although internally it does use them)